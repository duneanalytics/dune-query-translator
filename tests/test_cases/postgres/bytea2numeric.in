select bytea2numeric(data)