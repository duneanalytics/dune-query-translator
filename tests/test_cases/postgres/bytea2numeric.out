SELECT
  BYTEARRAY_TO_BIGINT(data)